chirpstack/src/storage/gateway.rs

```diff
@@ -308,17 +327,33 @@ pub async fn get_meta(gateway_id: &EUI64) -> Result<GatewayMeta, Error> {
     Ok(meta)
 }
 
+#[cfg(feature = "postgres")]
 pub async fn get_counts_by_state(tenant_id: &Option<Uuid>) -> Result<GatewayCountsByState, Error> {
     let counts: GatewayCountsByState = diesel::sql_query(r#"
         select
-            coalesce(sum(case when last_seen_at is null then 1 end), 0) as never_seen_count,
```

Comment:
The `coalesce` is needed because else it still might return `NULL` and you will see an error in the web-interface.  I will push a fix.